src/CoreText/ConstructorError.cs

```diff
@@ -39,7 +42,7 @@ static class ConstructorError {
 		public static Exception ArgumentNull (object self, string argument)
 		{
 			GC.SuppressFinalize (self);
-			return new ArgumentNullException (argument);
+			return ObjCRuntime.ThrowHelper.ThrowArgumentNullException (nameof (argument));
```

Comment:
This won't compile. ObjCRuntime.ThrowHelper.ThrowArgumentNullException returns void, not the exception because it throws.

Reply:
Oops good call! I didn't commit this change back!